Library/Formula/pop2imap.rb

```diff
@@ -0,0 +1,58 @@
+class Pop2imap < Formula
+  desc "POP to IMAP sync or copy tool."
+  homepage "http://www.linux-france.org/prj/pop2imap/pop2imap"
```

Comment:
This URL just points to the source code of the main `pop2imap` script. There doesn't seem to be a human-readable project home page per se. Maybe the README at http://www.linux-france.org/prj/pop2imap/README would be a better choice.